apps-rendering/src/components/PinnedPost/index.tsx

```diff
@@ -0,0 +1,208 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import {
+	background,
+	border,
+} from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { timeAgo } from '@guardian/libs';
+import {
+	focusHalo,
+	from,
+	neutral,
+	remHeight,
+	remSpace,
+	textSans,
+	transitions,
+	visuallyHidden,
+} from '@guardian/source-foundations';
+import {
+	SvgMinus,
+	SvgPinned,
+	SvgPlus,
+} from '@guardian/source-react-components';
+import { LiveBlock } from 'components/LiveBlocks';
```

Comment:
It's maybe worth noting that this introduces a circular/cyclic import. `PinnedPost` imports from `LiveBlocks` and `LiveBlocks` imports from `PinnedPost`. @iainjchambers-guardian and I had a conversation about circular imports recently, but I'm not sure whether we came to a conclusion on whether they might cause problems or not 🤔

I've mentioned both of you in a separate conversation about this to discuss. If you _did_ want to change this to avoid the circular import you could consider making `LiveBlock` a standalone component and have both `PinnedPost` and `LiveBlocks` import from it.

Reply:
Thanks Jamie, just saw your message. Yes great point. I think it'll be better and clearer if the LiveBlock is moved to be a standalone component. We'll make this change now. But please share with us later if you learn more how/if circular dependancy is going to be problematic